src/components/ModalDelete.js

```diff
@@ -0,0 +1,26 @@
+import Modal from "./Modal.js";
+
+export default class ModalDelete extends Modal {
+  constructor({ modalSelector, cardRemovalHandler }, card) {
+    super({ modalSelector });
+    this._deleteButton = document.getElementById("delete-image-confirm");
+    this._cardRemovalHandler = cardRemovalHandler;
+    this.card = card;
+  }
+  open(card) {
+    super.open();
+    this._deleteButton.innerText = "Yes";
```

Comment:
`    this._deleteButton.innerText = "Yes";
`

Please, delete it

You don't need it here anymore

Reply:
Thanks for catching this! I just changed it on the main branch.